content/en/tracing/guide/ignoring_apm_resources.md

```diff
@@ -233,16 +233,17 @@ Some of the language-specific tracers have an option to modify spans before they
 
 The Ruby tracer has a post-processing pipeline that deletes traces that meet certain criteria. More information and examples can be found in [Post-processing traces][1].
 
-For example, if the resource name is `Api::HealthchecksController#index`, use the `trace.delete_if` method to delete traces that contain the resource name. This filter can also be used to match on other metadata available for the [span object][2].
+For example, if the resource name is `Api::HealthchecksController#index`, use the `trace.spans.delete_if` method to delete traces that contain the resource name. This filter can also be used to match on other metadata available for the [span object][2].
 
-```
-Datadog::Tracing::Pipeline.before_flush do |trace|
-  trace.delete_if { |span| span.resource =~ /Api::HealthchecksController#index/ }
```

Comment:
I don't think this example is correct. It would delete only one span (a child span) from the trace. The trace will still come through, just without the one span, or any of its descendants.

I assume the intent is to filter out unwanted traces. Unfortunately, this behavior is broken right now as [detailed here](https://github.com/DataDog/dd-trace-rb/issues/2038#issuecomment-1137696303).

Best we can offer as a workaround right now is setting a `before_action` in the Rails controller that flags the active trace to not be sampled. e.g. `Datadog::Tracing.active_trace.reject!`.